tests/integration/components/callout-banner-test.js

```diff
@@ -0,0 +1,32 @@
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+module('Integration | Component | callout-banner', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(hbs`
+      <CalloutBanner>
+        I have an announcement to make
+      </CalloutBanner>
+    `);
+
+    assert
+      .dom('.callout-banner')
+      .exists()
+      .hasText('I have an announcement to make');
+  });
+
+  test('callouts can be made dismissible', async function (assert) {
+    await render(hbs`
+      <CalloutBanner @dismissible={{true}}>
+        You can dismiss this announcement
+      </CalloutBanner>
+    `);
+    assert.dom('.callout-banner').exists();
+    await click('.toggle-close');
+    assert.dom('.callout-banner').doesNotExist();
```

Comment:
Let's prefer a `data-test-*` over a CSS class. I also recommend providing an easy-to-read description for each assertion, to help other contributors understand what scenario the assertions cover.

```suggestion
    assert
      .dom('[data-test-callout-banner]')
      .exists('The user sees the callout banner.');

    await click('[data-test-button="Close"]');

    assert
      .dom('[data-test-callout-banner]')
      .doesNotExist('The user should not see the callout banner.');
```